rental.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# The Rental class represents a rental transaction, which includes a date, a book, and a person who rented the book.
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+  end
+
+  attr_accessor :date
+  attr_reader :person, :book
+end
```

Comment:
- [ ] A rental belongs to a `book` and also to a `person`. Please implement the `belongs-to` relationship between:
1. `Rental` and `Book`
2. `Rental` and `Person`
![image](https://user-images.githubusercontent.com/46839548/225502088-24268b35-f400-4e0c-9904-50694843341b.png)
![image](https://user-images.githubusercontent.com/46839548/225503407-f9b395c5-2d6c-4d98-be9c-37b7314e6262.png)